### ISSUE TITLE
%TWITTERCARD% not being replaced in exports

### ISSUE THREAD

**Issue by JohnSmith-LT** [thexerteproject/xerteonlinetoolkits#1124]
Not sure how long this has been in the wild but %TWITTERCARD% is not being replaced in any of the exports except for the Scorm 1.2 export.

I'll fix this when I update the 5x exports with the logo code but just checking @torinfo @ronm123 @FayCross that  the 3x exports (deployment, archive, offline) should have the TWITTERCARD activated (if included) but the 2x Scorm and xAPI should have it left blank?

**Comment by torinfo** [comment]
@JohnSmith-LT TWITTERCARD is new to develop, so not in 3.10.5 yet. Adn I also noticed that the %TWITTERCARD% was not replaced in all cases,but did not have the time yet to fix it. 

I suppose that it could be usefull on export, but @ronm123 is better placed to answer that.

I think you observation is correct: It has not use for the xapi, lti, and two flavours of scorm.

**Comment by JohnSmith-LT** [comment]
Thanks @torinfo. Leave that with me then - I'll replace it deployment, archive, offline and clear it in the rest when I update the logo code in these files.

**Comment by ronm123** [comment]
@JohnSmith-LT @torinfo We have discussed this previously and as you say Tom you was going to have a look at where else the code was needed to replace %TWIITERCARD% either with the card code or just removing the placeholder.

The trouble is it's not an exact science and is a bit more nuanced...
For the Twitter cards to work the LO needs to be public facing. That is very unlikely with SCORM and LTI because both require authentication so in both those cases I think the placeholder should be removed so it doesn't display.
Play obvious needs to work and perhaps xAPI in groupmode without LTI.
Deployment export should probably include the Twitter card but perhaps not offline export.
Then again this is all just metadata so perhaps it doesn't matter and perhaps all methods and exports should include the card info and whether it works or not is up to the author and how they make the project available.

**Comment by torinfo** [comment]
xApi in groupmode could be interesting, because it enable you to track at least the number of times it is used.

**Comment by JohnSmith-LT** [comment]
I don't think that there is time to implement any of this before this release so I've added code to just hide the %TWITTERCARD% placeholder on all the exports just now - I'll leave this issue open though because perhaps it needs revisiting by someone who makes use of the twittercards (not something that i've ever used to be honest)...
